src/partials/back_button.hbs

```diff
@@ -1,6 +1,14 @@
 <div class="step-link step-link--back" style="margin: 0; text-align: left;">
   <div style="margin: 0; text-align: left;">
-    <div class="back-button" id="changeDevice"><span
-            class="pingicon-left"></span></div>
+    <button id="changeDevice" class="button" style="width: 30px;
+                                                    height: 30px;
```

Comment:
Can we add this to main.scss? Might be easier for customers to customize there if needed